spec/API_specification/elementwise_functions.md

```diff
@@ -528,6 +528,10 @@ For floating-point operands,
 
 Calculates the division for each element `x1_i` of the input array `x1` with the respective element `x2_i` of the input array `x2`.
 
+```{note}
+If one or both of the input arrays have integer data types, the result is unspecified and thus implementation-dependent.
```

Comment:
Don't know if this needs a note or not, but if we're doing this, we should make this say "floating-point" https://github.com/data-apis/array-api/pull/362/files#diff-c63388022ee724b37174b4191598444df80aacb366d1f63677da4d14784df654R566

Reply:
Also for both divide and pow, if we do have a note would be clearer to have the note in the parameters section, so that it is right next to the part that says "numeric data types" or "floating-point data types".

Reply:
> Also for both divide and pow, if we do have a note would be clearer to have the note in the parameters section, so that it is right next to the part that says "numeric data types" or "floating-point data types".

Ah I agree that would make sense. For divide functions this seems tricky though, because the note also talks about the result dtype. Splitting the notes kinda loses the connectedness of the input and output.